src/sonarwhal-theme/layout/partials/home.hbs

```diff
@@ -22,86 +13,117 @@
-                    <h2 class="subheading subheading--home">Install sonarwhal</h2>
-                    <div class="layout layout--halves vertical-align">
-                        <div class="module">
-                            <img src="/images/home-nellie.svg" class="sub-illustration" alt="" role="presentation">
+        </section>
+        <div class="bg-wrap--white">
+            <section class="container home-container--section home-container__features">
+                <h2 class="subheading subheading--home">Why use sonarwhal?</h2>
+                <div class="layout layout--thirds">
+                    <div class="module">
+                        <img src="/images/nellie-customizable.svg" alt="" role="presentation" />
+                        <p class="subtitle subtitle--home uppercase-text">Fully customizable</p>
+                        <p class="feature__highlight">
+                            Every site is different. sonarwhal adapts its feedback when you give it more information:
+                            <a href="/docs/user-guide/#ignoringdomains">ignore 3rd-party code</a>,
+                            <a href="/docs/user-guide/#browserconfiguration">prioritize <em>your</em> users' browsers</a>, and <a href="/docs/user-guide/">control the
+                            results</a> with minimal setup.
+                        </p>
+                        <a class="feature__cta" href="/docs/user-guide/rules/">User guide</a>
+                    </div>
+                    <div class="module">
+                        <img src="/images/developer-nellie.svg" alt="" role="presentation" />
+                        <p class="subtitle subtitle--home uppercase-text">Create your own rules</p>
+                        <p class="feature__highlight">
+                            With the help of our <a href="/docs/contributor-guide/">developer guide</a>, you can
+                            <a href="/docs/contributor-guide/rules/index.html">create new rules</a> to suit <em>your</em> needs. You can
+                            help sonarwhal help even more people like you by contributing your rules back!
+                        </p>
+                        <a class="feature__cta" href="/docs/user-guide/rules/">Rule documentation</a>
+                    </div>
+                    <div class="module">
+                        <img src="/images/nellie-cookies.svg" alt="" role="presentation" />
+                        <p class="subtitle subtitle--home uppercase-text">Community driven</p>
+                        <p class="feature__highlight">
+                            sonarwhal welcomes anyone who wants to make the web a better place.
+                            Testing, <a href="https://github.com/sonarwhal/sonarwhal.com/issues">filing issues and feature requests</a>,
+                            <a href="/docs/developer-guide/contributing/pull-requests.html">contributing code</a>, and improving the documentation
```

Comment:
This should be: https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/